config/locales/ja.yml

```
@@ -0,0 +1,125 @@
+      default: "%Y年%m月%d日"
+      short: "%m/%d"
+      long: "%Y年%m月%d日(%a)"
+  
+  time:
+    formats:
+      default: "%Y年%m月%d日 %H:%M"
+      short: "%m/%d %H:%M"
+      long: "%Y年%m月%d日(%a) %H時%M分%S秒"
+  
+  # 曜日
+  date:
```

YAMLlint (1.37.1)

[error] duplication of key "date" in mapping

(key-duplicates)